node/src/chain_spec/battery_station.rs

```diff
@@ -115,7 +115,13 @@ pub(super) fn get_wasm() -> Result<&'static [u8], String> {
 
 generate_generic_genesis_function!(
     battery_station_runtime,
-    sudo: battery_station_runtime::SudoConfig { key: Some(root_key_staging_battery_station()) },
+    asset_registry: battery_station_runtime::AssetRegistryConfig {
+        assets: vec![],
+        last_asset_id: ForeignAsset(0),
```

Comment:
just brefely checked the code of orml asset-registry pallet, could not find how last_asset_id is being used.

Reply:
`last_asset_id` is not used at all in our implementation. If we would use the [`SequentialId`](https://github.com/open-web3-stack/open-runtime-module-library/blob/740f0b83f23f13dc540a8df2773b119446392c57/asset-registry/src/impls.rs#L26-L50) implementation, then it would be relevant. However, we don't impose any rules on the id (other than it's not used already): https://github.com/zeitgeistpm/zeitgeist/blob/e170adb979115d5794bcf2510acfe74a59146686/runtime/zeitgeist/src/xcm_config/asset_registry.rs#L31-L48

The reason why we allow any id is because the ids will be registered using governance, and you cannot allow guarantee that the order of referendums will be the order in which the proposals were made. For example, there could be proposals for id 1,2,3 , but governance takes up 2 as the first referendum, which will obviously fail if we enforce sequential ids. Furthermore a referendum could also be rejected, effectively rendering all consecutive asset registration requests invalid (i.e. referendum to onboard 1 got rejected, proposal to onboard 2,3, etc. automatically become invalid).